### PR TITLE
[refactoring] improve handling exceptions

### PR DESCRIPTION
**Pull request by asolntsev**
* throw "new UncheckedIOException(ioe)"" instead of just "new RuntimeException(ioe)"
* avoid catching too broad Exception which causes unneeded re-wrapping of RuntimeExceptions

### Description
it's just a small refactoring: improving some of "catch Exception" blocks.

P.S. Replacing RuntimeException by UncheckedIOException doesn't explicitly solve any practical problem, it's just a better exception for wrapping IOException.

You Many coding standards recommend to avoid throwing too generic exceptions like `RuntimeException` or `Exception`. And `UncheckedIOException` is exactly for that: it's a "RuntimeException for wrapping IOException".
 
### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: java/test/org/openqa/selenium/remote/tracing/opentelemetry/TracerTest.java [hunk 1]
```diff
@@ -49,6 +38,22 @@
 import org.openqa.selenium.remote.tracing.Tracer;
 import org.openqa.selenium.testing.UnitTests;
 
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.common.AttributeKey;
```
Comment (pujagani) [inline java/test/org/openqa/selenium/remote/tracing/opentelemetry/TracerTest.java]: Optimizing imports can probably be a different PR. It will help review the core changes easily.

Comment (asolntsev) [reply java/test/org/openqa/selenium/remote/tracing/opentelemetry/TracerTest.java]: I agree, it's just IDEA did it automatically. :)

File: java/src/com/thoughtworks/selenium/webdriven/JavascriptLibrary.java [hunk 2]
```diff
@@ -102,7 +103,7 @@ String readScriptImpl(String script) {
     try {
       return Resources.toString(url, StandardCharsets.UTF_8);
     } catch (IOException e) {
-      throw new RuntimeException(e);
+      throw new UncheckedIOException(e);
```
Comment (diemol) [inline java/src/com/thoughtworks/selenium/webdriven/JavascriptLibrary.java]: What is the reasoning behind this change? Could you please explain a bit more? Also in the PR description.

Comment (asolntsev) [reply java/src/com/thoughtworks/selenium/webdriven/JavascriptLibrary.java]: Hi @diemol !
Replacing `RuntimeException` by `UncheckedIOException` doesn't explicitly solve any practical problem, it's just a better exception for wrapping `IOException`. 

You Many coding standards recommend to avoid throwing too generic exceptions like `RuntimeException` or `Exception`. And `UncheckedIOException` is exactly for that: it's a "RuntimeException for wrapping IOException".

File: java/test/org/openqa/selenium/remote/tracing/opentelemetry/TracerTest.java (hunk 1 above)
Comment (asolntsev) [reply java/test/org/openqa/selenium/remote/tracing/opentelemetry/TracerTest.java]: @pujagani Now I have removed the change in imports.

File: java/src/com/thoughtworks/selenium/HttpCommandProcessor.java [hunk 3]
```diff
@@ -46,8 +46,8 @@
 public class HttpCommandProcessor implements CommandProcessor {
 
   private String pathToServlet;
-  private String browserStartCommand;
-  private String browserURL;
+  private final String browserStartCommand;
```
Comment (diemol) [inline java/src/com/thoughtworks/selenium/HttpCommandProcessor.java]: Can you please revert changes to files in the `thoughtworks` package? This is legacy code and we will eventually phase out RC.

File: java/src/com/thoughtworks/selenium/webdriven/JavascriptLibrary.java [hunk 4]
```diff
@@ -24,6 +24,7 @@
 import org.openqa.selenium.WebElement;
 
 import java.io.IOException;
+import java.io.UncheckedIOException;
```
Comment (diemol) [inline java/src/com/thoughtworks/selenium/webdriven/JavascriptLibrary.java]: Can you please revert changes to files in the `thoughtworks` package? This is legacy code and we will eventually phase out RC.

File: java/src/org/openqa/selenium/net/UrlChecker.java [hunk 5]
```diff
@@ -130,9 +130,7 @@ public void waitUntilUnavailable(long timeout, TimeUnit unit, final URL url)
       throw new TimeoutException(String.format(
           "Timed out waiting for %s to become unavailable after %d ms",
           url, MILLISECONDS.convert(System.nanoTime() - start, NANOSECONDS)), e);
-    } catch (RuntimeException e) {
-      throw e;
-    } catch (Exception e) {
+    } catch (InterruptedException | ExecutionException e) {
```
Comment (diemol) [inline java/src/org/openqa/selenium/net/UrlChecker.java:133]: If we are handling `InterruptedExcption`, shouldn't be the thread interrupted here?

File: java/test/com/thoughtworks/selenium/CSVTest.java [hunk 6]
```diff
@@ -22,6 +22,7 @@
 import org.junit.Before;
 import org.junit.Test;
 
+import java.lang.reflect.InvocationTargetException;
```
Comment (diemol) [inline java/test/com/thoughtworks/selenium/CSVTest.java]: Can you please revert changes to files in the `thoughtworks` package? This is legacy code and we will eventually phase out RC.

**Review by diemol — CHANGES_REQUESTED** [review]

File: java/src/com/thoughtworks/selenium/HttpCommandProcessor.java (hunk 3 above)
Comment (asolntsev) [reply java/src/com/thoughtworks/selenium/HttpCommandProcessor.java]: ok, I will revert.

Comment (asolntsev) [reply java/src/com/thoughtworks/selenium/HttpCommandProcessor.java]: @diemol Done (reverted changes in `com.thoughtworks` package).

File: java/src/com/thoughtworks/selenium/webdriven/JavascriptLibrary.java (hunk 4 above)
Comment (asolntsev) [reply java/src/com/thoughtworks/selenium/webdriven/JavascriptLibrary.java]: done

File: java/test/com/thoughtworks/selenium/CSVTest.java (hunk 6 above)
Comment (asolntsev) [reply java/test/com/thoughtworks/selenium/CSVTest.java]: done

File: java/src/org/openqa/selenium/net/UrlChecker.java (hunk 5 above)
Comment (asolntsev) [reply java/src/org/openqa/selenium/net/UrlChecker.java:133]: @diemol  Good point. Probably we should interrupt the current thread, yes. 
But... the thread was not interrupted before my change, and I just did a refactoring, so the code behaves exactly the same after my changes. :)

**Review by diemol — APPROVED** [review]
